requirements-python3/dev-requirements.txt

```diff
@@ -198,7 +198,7 @@ sphinxcontrib-htmlhelp==1.0.2  # via sphinx
 sphinxcontrib-jsmath==1.0.1  # via sphinx
 sphinxcontrib-qthelp==1.0.2  # via sphinx
 sphinxcontrib-serializinghtml==1.1.3  # via sphinx
-sqlagg==0.16.2            # via -r requirements/requirements.in
+sqlagg==0.17.0b0          # via -r requirements/requirements.in
```

Comment:
would we not push a proper release before adding it as an HQ dependency? (as opposed to a beta)

Reply:
indeed - we'll actually need to update this to `0.17.2` since there have been some other changes done since then. @mkangia can you update the version please

Reply:
Sure, i will bump that up.